CHANGELOG.next.asciidoc

```diff
@@ -91,7 +91,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 ==== Bugfixes
 
 *Affecting all Beats*
-
+- common/transport: Fix TLS status message from `test output` command. {pull}30063[30063]
```

Comment:
Please do not include the package names in the changelog entry. This changelog is written for our users, they should not know about where the log message is located in Beats. I really like how you put it in the PR description, I would like you to explain the change similarly. Example: When it `ssl.verification_mode` was set to `full` or `certificate`, the command `test output` logged incorrectly that TLS was disabled.